.bazelversion

```diff
@@ -1 +1 @@
-6.4.0
\ No newline at end of file
+6.5.0
```

Comment:
curious why we need to bump this?

Reply:
Some of the referenced Java dependencies are not available, and the dependent source code of tcl can not be downloaded  from the sourceforge.

153.7 WARNING: Download from https://mirror.bazel.build/repo1.maven.org/maven2/com/google/guava/guava/32.1.1-jre/guava-32.1.1-jre.jar failed: class java.io.FileNotFoundException GET returned 404 Not Found
153.9 Analyzing: 3976 targets (354 packages loaded, 27976 targets configured)
154.9 Analyzing: 3976 targets (358 packages loaded, 28002 targets configured)
155.9 Analyzing: 3976 targets (360 packages loaded, 28022 targets configured)
156.9 Analyzing: 3976 targets (361 packages loaded, 28047 targets configured)
158.0 Analyzing: 3976 targets (362 packages loaded, 28047 targets configured)
159.0 Analyzing: 3976 targets (367 packages loaded, 29507 targets configured)
159.6 WARNING: Download from https://mirror.bazel.build/repo1.maven.org/maven2/com/google/errorprone/error_prone_annotations/2.23.0/error_prone_annotations-2.23.0.jar failed: class java.io.FileNotFoundException GET returned 404 Not Found
159.8 WARNING: Download from https://mirror.bazel.build/repo1.maven.org/maven2/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar failed: class java.io.FileNotFoundException GET returned 404 Not Found

Reply:
Following Hzeller's suggestion, I tried 6.5.0, and it works.